frontend/src/pages/Demo/DemoTailwind.tsx

```diff
@@ -1,14 +1,79 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Typography from "tw-components/Typography";
+import { Button, IconButton } from "tw-components/Buttons";
 
 const DemoTailwind = () => {
+  // Add a setDarkMode for testing dark mode styles
+  const [darkMode, setDarkMode] = useState(false);
+
+  useEffect(() => {
+    if (darkMode) {
+      document.documentElement.classList.add("dark");
+    } else {
+      document.documentElement.classList.remove("dark");
+    }
+  }, [darkMode]);
+
   return (
     <div className="min-h-screen bg-grey-light p-8">
+      <div className="mx-auto max-w-4xl rounded-lg bg-white p-6 shadow-md dark:bg-[#030D2D] transition-colors duration-300">
```

Comment:
```suggestion
      <div className="mx-auto max-w-4xl rounded-lg bg-white p-6 shadow-md dark:bg-[#030D2D] transition-colors duration-300">
```
If you dont already have it, please get eslint extension on vscode and configure it automatically lint your frontend files.

There is a linter rule for tailwind classnames order that some lines in this file are throwing an error for. You can check by going to the `/frontend/` folder and running `npm run lint`. It should also automatically fix them for you 👍